actors/test/actor-tests/src/main/java/cloud/orbit/actors/test/FakeClock.java

```diff
@@ -66,7 +66,7 @@ public long millis()
     {
         if (stopped)
         {
-            return millis.incrementAndGet();
+            return millis.get();
```

Comment:
@ben-manes fyi this was the root cause of the failure for https://github.com/orbit/orbit/pull/144#issuecomment-204718574